nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/clear_costmap_service.hpp

```diff
@@ -19,6 +19,8 @@
 
 #include "nav2_behavior_tree/bt_service_node.hpp"
```

Comment:
https://app.circleci.com/pipelines/github/ros-planning/navigation2/4713/workflows/45fd3136-d3a2-47d7-b7f4-1f161b1fbef1/jobs/17524 some linting errors, you can run these locally with `ament_cpplint` and `ament_uncrustify` so you can see these before CI runs

Reply:
Thx for the tips